antlr/src/main/antlr/FSH.g4

```diff
@@ -102,8 +124,11 @@ NUMBER:             [+\-]? [0-9]+('.' [0-9]+)?;
                  //   '  UCUM UNIT   '
 UNIT:               '\'' (~[\\'])* '\'';
 
-                 // SYSTEM     #  SYSTEM   "DISPLAY"
-CODE:               SEQUENCE? '#' SEQUENCE STRING?;
+                 // SYSTEM     #  SYSTEM
+CODE:               SEQUENCE? '#' (SEQUENCE | CONCEPT_STRING);
```

Comment:
Since you changed the existing `CODE` rule here, can you update the importer to accommodate the change?  Basically, anywhere that parses a rule that uses the `CODE` token, it will need to check if the resulting code value (after `#`) is wrapped in `"`.  If so, it will want to:
* remove surrounding `"`
* unescape any `\"`
* unescape any `\\`